scripts/firefox/jumpcloud/windows/v1/zluri-windows-firefox-removal-jumpcloud.ps1

```diff
@@ -0,0 +1,61 @@
+$removalSuccess = $true
+
+$fireFoxPathJson = Get-Content -Raw "C:\Program Files\Mozilla Firefox\distribution\policies.json" | ConvertFrom-Json
+
+if ($fireFoxPathJson.policies.ExtensionSettings.PSObject.Properties.Name -contains "zluribrowseragent@zluri.com") {
+    Write-Output "Removing Zluri Firefox extension from policies.json"
+    Remove-Item "C:\Program Files\Mozilla Firefox\distribution\policies.json" -Force -ErrorAction SilentlyContinue
+    if (Test-Path "C:\Program Files\Mozilla Firefox\distribution\policies.json") {
+        Write-Output "Failed to remove policies.json"
+        $removalSuccess = $false
+    }
+} else {
+    Write-Output "Zluri Firefox extension doesn't exist in policies.json"
+}
+
+$profilesPath = "C:\Users\TRACK\AppData\Roaming\Mozilla\Firefox\Profiles"
+
+if (Test-Path $profilesPath) {
+    $profiles = Get-ChildItem -Path $profilesPath -Directory
+
+    foreach ($profile in $profiles) {
+        $extensionsPath = Join-Path $profile.FullName "extensions"
+
+        if (Test-Path $extensionsPath) {
+            $extensionFiles = Get-ChildItem -Path $extensionsPath -File
+
+            foreach ($file in $extensionFiles) {
+                if ($file.Name -like "*zluribrowseragent@zluri.com.xpi*") {
+                    Write-Output "Removing extension $($file.FullName) from profile $($profile.Name)"
+                    
+                    try {
+                        # Attempt to remove the file
+                        Remove-Item -Path $file.FullName -Force -ErrorAction Stop
+                        # Verify if the file still exists
+                        if (Test-Path $file.FullName) {
+                            Write-Output "Failed to remove extension $($file.FullName) - File still exists."
```

Comment:
@sashank-zluri @Dhanush-akrity Callout here
This function will only work if the firefox extension is not open on the machine
While testing I saw we get "Access Denied" error for this statement if the firefox browser is open

Reply:
@ashish-zluri Please check how we can force quit this.

Reply:
We shouldn't force quit ig. 

Quit with an option ?  

Force quit isnt a viable option IMO.

Reply:
Force quit extension. Not browser

Reply:
@dhanush-zluri How to force quit extension ?

Reply:
Even though in our scripts we are not even adding the extension to this path the extension is getting added here
Force quit browser is one option I see, but that isn't a good user experience.

Should be mark this as callout itself in our docs that browser should be closed when doing this step ?